fedimint-recurringd/src/lib.rs

```diff
@@ -71,12 +71,16 @@ impl RecurringInvoiceServer {
             clients.insert(federation_id, Arc::new(client));
         }
 
-        Ok(Self {
-            db,
+        let slf = Self {
+            db: db.clone(),
```

Comment:
Weird that this clone is needed, as there seem to be nothing using db till the end of scope. And I wonder if clippy should complain about it. Or maybe I just need more coffee.

Reply:
Good catch, wondering why clippy didn't catch it

Reply:
(Why it was there originally: I was silly and redundantly supplied `db` to `run_db_migrations`)